docs/nodes/mamaki.md

```diff
@@ -55,12 +66,14 @@ cp $HOME/networks/mamaki/genesis.json $HOME/.celestia-app/config
 Set seeds and peers:
 
 ```sh
-SEEDS="74c0c793db07edd9b9ec17b076cea1a02dca511f@46.101.28.34:26656"
-PEERS="34d4bfec8998a8fac6393a14c5ae151cf6a5762f@194.163.191.41:26656"
+SEEDS="f0c58d904dec824605ac36114db28f1bf84f6ea3@144.76.112.238:26656"
+PEERS="e4429e99609c8c009969b0eb73c973bff33712f9@141.94.73.39:43656"
```

Comment:
we ideally add more peers here from our peers.txt list in the networks/mamaki folder